bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,154 @@
+| powerSelfSupplied | Number:Energy | Energy consumed direct from PV plus energy consumed from storage
+| powerConsumedFromGrid | Number:Energy | Energy consumed from the grid  
+| powerConsumedFromStorage | Number:Energy | Energy consumed from storage
+| powerConsumed | Number:Energy | Total energy consumed. All inner and outer consumers.
+| powerProduced | Number:Energy | Energy produced by the PV
+| powerOut | Number:Energy | Energy delivered to the grid  
+| powerDirectConsumed | Number:Energy | Energy consumed directly without energy put into storage or taken from storage
+| workBuffered | Number:Energy | Energy flow into the storage system
+| workSelfConsumed | Number:Energy | Energy consumed direct from PV plus energy stored
+| workSelfSupplied | Number:Energy | Energy consumed direct from PV plus energy consumed from storage
+| workConsumedFromGrid | Number:Energy | Energy consumed from the grid  
+| workConsumedFromStorage | Number:Energy | Energy consumed from storage
+| workConsumed | Number:Energy | Total energy consumed. All inner and outer consumers.
+| workProduced | Number:Energy | Energy produced by the PV
+| workOut | Number:Energy | Energy delivered to the grid  
+| workDirectConsumed | Number:Energy | Energy consumed directly without energy put into storage or taken from storage
+
+### PowerMeter, S0Counter, MyReservePowerMeter
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| channelDirectionMetering | String | Representing which energy flow directions are metered. One off *IN*, *OUT*, *BIDIRECTIONAL* 
+| powerIn | Number:Energy | Energy metered flowing into the consumer
+| powerOut | Number:Energy | Energy metered flowing out of the producer  
+| workIn | Number:Energy | Energy metered flowing into the consumer
+| workOut | Number:Energy | Energy metered flowing out of the producer  
+| consumptionEnergySum | Number:Energy | Total energy in watt hours 
+
+### Inverter, MyReserveInverter, SunSpecInverter
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| powerACOutMax | Number:Energy | Maximum power production 
+| powerACOutLimit | Number:Energy | Limit of power production
+| powerACOut | Number:Energy | Energy delivered by the inverter
+| workACOut | Number:Energy | Energy delivered by the inverter
+| powerInstallledPeak | Number:Energy | Technical peak power available 
+
+### BatteryConverter, MyReserve
+
+All of *Inverter* plus
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| powerACIn | Number:Energy | Energy fed into battery
+| workACIn | Number:Energy | Energy fed into battery
+| stateOfCharge | Number | Charging state of battery in percent
+| stateOfHealth | Number | Internal health metric in percent
+| temperatureBattery | Number:Temperature | Temperature of the battery in celsius
+| modeConverter | String | Current mode of converter. *ON* or *OFF*
```

Comment:
You could make this a `Switch` Item.

Reply:
Even if it is nothing that we can switch on or off by a command?

Reply:
That's no problem. You can use a `Switch` for a read-only Channel.